package/Runtime/Plugins/iOS/NimbusBinding.mm

```diff
@@ -0,0 +1,76 @@
+//
+//  NimbusBinding.mm
+//
+//  Created by Bruno Bruggemann on 5/7/21.
+//  Copyright © 2021 Timehop. All rights reserved.
+//
+
+#import "NimbusSDK-Swift.h"
+
+#pragma mark - Helpers
+
+// Converts C style string to NSString
+#define GetStringParam(_x_) ((_x_) != NULL ? [NSString stringWithUTF8String:_x_] : [NSString stringWithUTF8String:""])
+#define GetNullableStringParam(_x_) ((_x_) != NULL ? [NSString stringWithUTF8String:_x_] : nil)
+
+#pragma mark - C interface
+
+extern "C" {
+
+    void _initializeSDKWithPublisher(const char* publisher,
+                                     const char* apikey,
+                                     bool enableSDKInTestMode,
+                                     int logLevel,
+                                     const char* appName,
+                                     const char* appDomain,
+                                     const char* bundleId,
+                                     const char* storeUrl,
+                                     bool showMuteButton) {
+        NSString* publishString = GetStringParam(publisher);
+        NSString* apiKeyString = GetStringParam(apikey);
+        NSString* appNameString = GetStringParam(appName);
+        NSString* appDomainString = GetStringParam(appDomain);
+        NSString* bundleIdString = GetStringParam(bundleId);
+        NSString* storeUrlString = GetStringParam(storeUrl);
```

Comment:
nice remember this is all going away, the only thing we will need to pass the to SDK will be apikey becauase both Android and iOS are making those fields optional since I know store them all in the DB

Reply:
Ok, I will change it now